dist/mapdc.js

```diff
@@ -7858,7 +7858,10 @@ var momentUTCFormat = exports.momentUTCFormat = function momentUTCFormat(d, f) {
   return _moment2.default.utc(d).locale("en").format(f);
 };
 var genericDateTimeFormat = exports.genericDateTimeFormat = function genericDateTimeFormat(d) {
-  return momentUTCFormat(d, "MMM D, YYYY") + " \u205F" + momentUTCFormat(d, "HH:mm:ss");
+  if (d.getMilliseconds() === 0) {
```

Comment:
For now, we're truncating millisecond timestamps is the `d.getMilliseconds() === 0`